src/configuration/prima.rs

```diff
@@ -6,8 +6,11 @@ use crate::TagTrackerConfiguration;
 use std::fmt::Display;
 use std::str::FromStr;
 
+/// By binding to 0.0.0.0:0 we're just letting the OS assign us a port, and letting anyone send us UDP packets on that port
+const DEFAULT_FROM_ADDR: &str = "0.0.0.0:0";
+
 /// The struct that represents options for the Datadog client in Prima.
-pub struct PrimaConfiguration {
```

Comment:
I think that what makes this config a `Prima` config is that we are enforcing the `env`. It's not said that the environment is needed and that should be one of the entry